Revans fun houses

AnonymousRevan opened the pull request.

Narizna commented:
for clarification - this is an AAT PR (after we update).

Changes talked about here are mainly the following
1. Holland becoming the "5th allied minor" slot instead of New Zealand.
2. Holland becoming the "annex allied AI minors" nation - to alleviate supply issues that have been found wanting due to previous changes
3. Mexico adjustments to prepare it for playability

I am assuming something about bulgaria - but I think we agreed to withhold that until hungary-romania interaction with 2nd vienna award and territory control

AnonymousRevan commented:
Mexico
Starting eco 18civs, 8mils, 1dock to 16civs, 6mils, 2docks
Refinery build speed to all refineries on left oil path lowered from 15% to 7.5%
Construction speed on right oil path lowered from 10% to 5%
Infra and civ speed lowered on right political path from 15% civ speed and 10% infra speed to 10% civ speed and 5% infra speed
Lowered spawn in divs on right political path from 8 to 3 divs.
Changed superior firepower theorist to regular military theorist
Loses rubber building debuff when Japan goes to war with the Allies.

Holland
Can annex Yugo, Belgium, Luxembourg, Greece, Norway, Poland, and Denmark once those nations cap.
Belgium Congo goes to the Dutch after Belgium caps. Holland gets cores on the Congo

Raj
Removed malus to medium tank production if you go green path. Lose it when you take the British tank focus.
British tank focus can now be taken on Sept 1, 1939 instead of June 1, 1940.

England
Now unlocks Secretary of State for the Home Department by completing Force Abdication focus
Modernize home industry now modifies Bilateral Trade Agreements idea by making the -5% consumer goods permanent along with 5% civ building speed.

SAF
Fixed decisions SAF had to put ships into production

Misc
Yugo, Turkey, Greece, Poland, and Norway no longer start with any ships.

Narizna commented:
due to the branch being created before finishing the AAT branch - there are some conflicts, if they can be resolved then I can approve
